src/lib/GovernorTimelockCompound.sol

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockCompound.sol)
+
```

Comment:
I think we should add a comment here at the top of this file explaining why we have it at all. That is, explain that the PoolTogether Timelock (and token? or is that not relevant here?) have a different interface than a standard GovernorAlpha fork, and as a result, the OpenZeppelin contracts would not work out of the box, and had to be updated to change the interfaces. We should make it clear that only function signatures were changed (is that true) and include a permalink to the OpenZeppelin commit where the files were forked from.

Reply:
Yes, will definitely do. I think the only non-signature change was we changed a method call from `GRACE_PERIOD` to `gracePeriod`. I am going to go through and document more thoroughly.

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/13/commits/155134d3c1719d583cd0eb425e53c1d86852c5b8